### PR TITLE
added windoge98 light theme

### PR DESCRIPTION
**Pull request by nolyoi**
Added a Windoge98 theme. If you guys have any suggestions on how I could improve it, I would like to hear.

### CODE REVIEW

**Review by julianjelfs — COMMENTED** [review]
Looking forward to taking a look at this. I will try to get to it soon.

**Comment by julianjelfs** [comment]
In the meantime @nolyoi - is there anything that you would have liked to be able to control that you couldn't control when you were trying to make is theme? Would like the make the theme system more flexible where necessary if I can.

File: frontend/app/src/theme/community/windoge98.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+export function getTheme(base: Theme): Theme {
+    base.author = "rh2pm-ryaaa-aaaan-qeniq-cai";
+    base.name = "windoge98";
+    base.label = "Windoge98";
+    base.bg = "#008080 url('/assets/windoge98logo.png') bottom 60px right 10px no-repeat";
+    base.burst = false;
+    base.logo = false;
+    base.txt = txt80;
+    base["txt-light"] = txt60;
+    base.bd = silver;
```
Comment (julianjelfs) [inline frontend/app/src/theme/community/windoge98.ts:30]: In some cases the borders cannot be seen using this colour. I would suggest something a whisker darker like #b0b0b0 maybe.

File: frontend/app/src/theme/community/windoge98.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    base.card.rd = "0";
+    base.modal.filter = "blur(5px)";
+    base.modal.bg = silver;
+    base.modal.bd = silver;
+    base.modalPage.bg = trolleyGray;
+    base.modalPage.txt = txt;
+
+    base.currentChat.msg.bd = trolleyGray;
+    base.currentChat.msg.bg = silver;
```
Comment (julianjelfs) [inline frontend/app/src/theme/community/windoge98.ts:85]: I would consider adding: 

```
    base.currentChat.msg.r1 = "0";
    base.currentChat.msg.r2 = "0";
```

File: frontend/app/src/theme/community/windoge98.ts [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    base.menu.warn = errorRed;
+
+    base.button["disabled-txt"] = disabledTxt;
+    base.button.bg = msBlue;
+    base.button.hv = magenta;
+
+    base.card.rd = "0";
+    base.modal.filter = "blur(5px)";
+    base.modal.bg = silver;
+    base.modal.bd = silver;
```
Comment (julianjelfs) [inline frontend/app/src/theme/community/windoge98.ts:80]: I would consider adding: 

```
    base.modal.rd = "0";
```

File: frontend/app/src/theme/community/windoge98.ts [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    base.icon.inverted.txt = black;
+    base.icon.selected = white;
+    
+    base.recommended.bg = white;
+    base.markdown.fg.color = txt;
+    base.markdown.fg.bright = txt;
+    base.markdown.fg.muted = txt40;
+
+    base.vote.yes.color = successGreen;
+    base.vote.no.color = errorRed;
```
Comment (julianjelfs) [inline frontend/app/src/theme/community/windoge98.ts:104]: I would consider adding: 

```
    base.notificationBar.bg = magenta;
    base.notificationBar.txt = white;
```

At the moment the notification bar (and the thread summary) will appear orange (default) which looks a bit out of place.

File: frontend/app/src/theme/community/windoge98.ts [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+const txt80 = hexPercent(txt, 80);
+const disabledTxt = txt40;
+const white = "#FDFFFF";
+const black = "#000000";
+const silver = "#C0C0C0";
+const trolleyGray = "#818181";
+const msBlue = "#010081";
+const darkGray = "#696969";
+const magenta = "#ff0081";
+const teal = "#008080";
```
Comment (julianjelfs) [inline frontend/app/src/theme/community/windoge98.ts:17]: This `teal` value is unused (which will break the build) so should be removed.

**Review by julianjelfs — COMMENTED** [review]
A few small suggestions. It's very close though. Love it.

**Comment by nolyoi** [comment]
> In the meantime @nolyoi - is there anything that you would have liked to be able to control that you couldn't control when you were trying to make is theme? Would like the make the theme system more flexible where necessary if I can.

Yeah it wasn't too bad, just lacking a bit of granularity between the panels. For example, the icon color is like a global per theme, so mixing some lighter and darker colors can be a challenge. Same with the `base.chatSummary["bg-selected"]/hv`.

Another thing I wanted to do was remove some more of the rounding to make it more retro looking, but wasn't able to tweak the `currentChat` rounding.

Having some different font choices would be pretty cool too.

I'll get some of your recommendations in there later on tonight after the holiday. Cheers!

File: frontend/app/src/theme/community/windoge98.ts (hunk 2 above)
Comment (nolyoi) [reply frontend/app/src/theme/community/windoge98.ts:85]: ah this is what I wanted!

**Review by julianjelfs — APPROVED** [review]
I have a couple of follow up changes that will allow us to remove some more of those curves, but I'd say let's get this in as is first.

**Pull request merged**